### DISCUSSION TITLE
How to extract password protected backups

### DISCUSSION THREAD

**Discussion by datbilling** [jcwillox/hass-auto-backup#84] [Q&A] [answered]
Is it possible to extract the contents of the encrypted backup tar file?

**Comment by jcwillox** [comment, marked as answer]
Technically yes, practically it is quite difficult, you'd need to write actual code to perform the decryption, as I'm not sure Home Assistant provides a tool to do that, and they don't use normal archive encryption so 7zip won't be able to do it.

I actually gave up on encrypting them with HA and just get rclone to encrypt my backups when it uploads them to google drive.
